Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Math/Vector2Nodes.h

```diff
@@ -243,6 +243,22 @@ namespace ScriptCanvas
         }
         SCRIPT_CANVAS_GENERIC_FUNCTION_NODE(ToPerpendicular, k_categoryName, "{CC4DC102-8B50-4828-BA94-0586F34E0D37}", "returns the vector (-Source.y, Source.x), a 90 degree, positive rotation", "Source");
 
+        AZ_INLINE void DirectionToDefaults(Node& node)
+        {
+            SetDefaultValuesByIndex<0>::_(node, Data::Vector2Type());
+            SetDefaultValuesByIndex<1>::_(node, Data::Vector2Type());
+            SetDefaultValuesByIndex<2>::_(node, Data::NumberType(1.));
+        }
+
+        AZ_INLINE Vector2Type DirectionTo(const Vector2Type from, const Vector2Type to, NumberType optionalScale = 1.f)
+        {
+            Vector2Type r = to - from;
+            r.Normalize();
+            r.SetLength(optionalScale);
+            return r;
+        }
+        SCRIPT_CANVAS_GENERIC_FUNCTION_NODE_WITH_DEFAULTS(DirectionTo, DirectionToDefaults, k_categoryName, "{49A2D7F6-6CD3-420E-8A79-D46B00DB6CED}", "Given two points in space, return a direction vector", "From", "To", "Scale");
```

Comment:
I suggest you mention it is normalized i.e. " return a normalized direction vector"
Another suggestion is to also return the distance since you're calculating it here anyways.  One of the common things I see in game logic is needing to find the direction and range, especially for AI tracking targets to know if say, an object is visible (dot product) and in vision or audio range.

Reply:
I do mention that it's normalized and optionally scaled in the translation file, but I will make this one match. I'll also add the distance, that's a good suggestion!